src/lib/tabs/tab-nav-bar/tab-nav-bar.ts

```diff
@@ -92,16 +93,28 @@ export class MdTabNav implements AfterContentInit, OnDestroy {
   }
 }
 
+
+// Boilerplate for applying mixins to MdTabLink.
+export class MdTabLinkBase {}
+export const _MdTabLinkMixinBase = mixinDisabled(MdTabLinkBase);
+
 /**
  * Link inside of a `md-tab-nav-bar`.
  */
 @Directive({
   selector: '[md-tab-link], [mat-tab-link], [mdTabLink], [matTabLink]',
-  host: {'class': 'mat-tab-link'}
+  inputs: ['disabled'],
+  host: {
+    'class': 'mat-tab-link',
+    '[class.mat-tab-disabled]': 'disabled'
```

Comment:
Also needs `aria-disabled` and setting `tabIndex` to `-1` (same as `MdAnchor`) with corresponding tests.